ipynb/.ipynb_checkpoints/metadata_template_generator_v2-checkpoint.ipynb

```diff
@@ -0,0 +1,217 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generate MIMS-compliant metadata template specific to environmental package"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import click\n",
+    "import pandas as pd\n",
+    "import re"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "qiita_ebi_mims_path = '../data/MIxS/Qiita_EBI_MIMS_v1.xlsx'  # required fields for Qiita, EBI, and MIMS (general)\n",
```

Comment:
Can you add the comments in a line above the actual command? Something like
```
# my comment
my code
```
I think it will be easier to read.